linting: only apply linting

PolynomialDivision opened the pull request.

PolynomialDivision commented:
@FFHener nur die letzten 3 sind rfc, die anderen sind für mich fertig. die letzten 3 würde ich also nicht mergen, wenn das hier approved ist.

PolynomialDivision commented:
For the rfc comments i still have to fix for the 802.11s mesh

```
config interface 'mesh_ap3'    
        option device 'wlan5-mesh'
        option proto 'none' 
```

to 

```
config interface 'mesh_ap3'    
        option device 'switch0.20'
        option proto 'none' 
```

PolynomialDivision commented:
I will merge this if no one objects?

Noki commented:
It's really a lot, therefore I only managed to have a quick look. Overall I like the changes, especially unifying spaces within brackets and unifying quotes and doublequotes. The only thing I do not like is loosing the formatting for IPs in the management section:

![image](https://github.com/user-attachments/assets/7d14a217-e0c1-41ca-9776-eb683b84c08d)

In my opinion the original was better, but that might be just personal preference, therefore no real objection.

Did you create configs for all locations before and after your changes and diff those and look for possible problems?

pktpls commented:
Bitte ein Makefile damit ich die check und fix steps lokal ausführen kann. Mir blutet jedes mal das Herz, wenn ich sehe wie jemand immer wieder editieren und pushen muss :-)

pktpls commented on locations/wuhlipunks.yml:
```diff
@@ -9,7 +9,6 @@ height: 5
 community: true
 
 hosts:
-
```
Diese Leerzeilen sind nicht erlaubt? Finde die manchmal ganz angenehm für die Übersichtlichkeit. (Ist in den locations natürlich jedem selbst überlassen.)

pktpls commented:
In falter-packages sieht das Makefile so aus: https://github.com/freifunk-berlin/falter-packages/blob/main/Makefile

- `make lint` macht shellcheck und luacheck
- `make fmt` macht shfmt

PolynomialDivision commented:
> Did you create configs for all locations before and after your changes and diff those and look for possible problems?

I looked at an ap, core and gateway-config.

I am not sure how I can do this with the tabs. :/ Is that a blocker?

Noki commented:
No blocker from my side...

PolynomialDivision commented:
I’ve just finished rebasing everything and reapplied the linting. I also added a Makefile. Once CI passes, I’ll go ahead and merge it. I'd prefer to avoid rebasing repeatedly. Hope that’s okay, as I’ve addressed all the feedback.
